src/hardware/tandy_sound.cpp

```diff
@@ -315,9 +315,7 @@ void TandyDAC::DmaCallback([[maybe_unused]] const DmaChannel*, DMAEvent event)
 
 void TandyDAC::ChangeMode()
 {
-	// Avoid under or overruning the mixer with invalid frequencies
-	// Typically frequencies are in the 8 to 22 Khz range
-	constexpr auto dac_min_freq_hz = 4900;
+	// Typical frequencies run up to 22 Khz range
 	constexpr auto dac_max_freq_hz = 49000;
```

Comment:
Bit confused where 49 Khz is derived from based on the comment. Actually, the 4.9k min freq is weird too.

Reply:
Yup, these were based on logging all the 'in the wild' Tandy DAC games that I knew of at the time, plus modern-ish '90s mod players that could play through the DAC.

These bounds sat just outside the actual ranges I saw. The Tandy DAC rates are set by dividing the PSG's NTSC-derived clock rate (~3.58 GHz) by a 12-bit divider.

Turns out _OutRun (1989)_ was a DAC game I didn't know about, and it uses a very large divider to stretch (and lower) the engine rumble at lower car speeds! Pretty neat.

See the updated second commit - added more comments, renamed 'freq' to 'sample_rate', and renamed the 'frequency' register to 'clock_divider'.